app/database/db-connection.service.js

```diff
@@ -0,0 +1,11 @@
+import ConfigService from '../config/config.service';
```

Comment:
database-access is a more appropriate name for the directory.

Also rename db-connection.service to mysql-connection.service. This is because we will be adding elastic search connection soon.